ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/TracingBlock.java

```diff
@@ -58,6 +63,55 @@
 
 public class TracingBlock {
 
+	/**
+	 * This class encapsulates all calls to the tracing options manager and runs them using a
+	 */
+	private static record TracingOptionsManagerDelegate(IRunnableContext fRunnableContext) {
+
+		Map<String, String> getTracingTemplateCopy() {
+			return runShowingProgress(
+					(tracingOptionsManager, monitor) -> tracingOptionsManager.getTracingTemplateCopy(monitor));
```

Comment:
If it's ok to you, I'd rather keep it as it is: I find method references with parameters a bit harder to read so I usually go for the "verbose" option. The current style also matches the style of the other 2 calls in the class so that's another reason to keep it like this.

Reply:
If you insist that's fine in this case :)